resources/k.json

```diff
@@ -29,8 +29,8 @@
         },
         "substate": {
           "selfDestruct": "VSelfDestruct",
-          "log": "_ => VLog",
-          "refund": "_ => VRefund"
+          "log": "_ => ?VLog",
+          "refund": "_ => ?VRefund"
```

Comment:
These should probably be changed to `_ => _` instead

Reply:
I think that might produce a warning too, asking it to be changed to `_ => ?_`. Does that sound plausible?

Reply:
@MrChico also, why does `_ => _` make more sense? Or why did `_ => VLog` make sense before?

Reply:
I think `_` on RHS is no problem. I think we should only really give names to RHS variables if we plan on making a claim on them. When we simply don't care about the RHS we should just keep it as `_`

Reply:
I don't think `_ => VLog` ever made much sense

Reply:
oh.. Now I see that there is a warning for `_` as well... hmm, I think this should be exempt

Reply:
we should keep these as `_` (without `=>`)